food-for-thought/app.json

```diff
@@ -31,6 +31,9 @@
     ],
     "experiments": {
       "typedRoutes": true
+    },
+    "extra": {
+      "HOST_IP": ""
```

Comment:
Maybe update readme to include the file direc and what to change?

Reply:
Otherwise LGTM

Reply:
Just pushed those changes! Markdown looks all good🙏
https://github.com/abrarbarbhuyia/2024SIS_Team5-/commit/8e9254f11ec12ef613de6bbe6b906234ed5e6e49